packages/components/src/standalone-comparison-grid/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@wordpress/components';
+import './style.scss';
+import { ReactNode } from 'react';
+
+interface StandAloneComparisonGridColumn {
+	title: string;
+	features: ReactNode[];
+	introCopy?: string;
+	actionCopy: string;
+	action: () => void;
+}
+
+export interface StandAloneComparisonGridProps {
+	children:
+		| React.ReactElement< typeof StandAloneComparisonGridColumn >
+		| React.ReactElement< typeof StandAloneComparisonGridColumn >[];
+}
+
+const StandAloneComparisonGridColumn = ( {
+	title,
+	introCopy,
+	action,
+	actionCopy,
+	features,
+}: StandAloneComparisonGridColumn ) => {
+	return (
+		<div className="standalone-comparison-grid__column standalone-comparison-grid__body">
+			<h2 className="standalone-comparison-grid__title">{ title }</h2>
+			<p className="standalone-comparison-grid__intro">{ introCopy }</p>
+			<Button className="standalone-comparison-grid__action" variant="primary" onClick={ action }>
+				{ actionCopy }
+			</Button>
+			<ul className="standalone-comparison-grid__row-list">
+				{ features.map( ( feature, index ) => (
+					<li key={ `scdi_${ index }` } className="standalone-comparison-grid__row-item">
+						{ feature }
+					</li>
+				) ) }
+			</ul>
+		</div>
+	);
+};
+
+export { StandAloneComparisonGridColumn };
+
+const StandAloneComparisonGrid: React.FC< StandAloneComparisonGridProps > = ( { children } ) => {
```

Comment:
```suggestion
export const StandAloneComparisonGrid: React.FC< StandAloneComparisonGridProps > = ( { children } ) => {
```


Since this package is exporting two components that could be (or not) used together the idea of the default doesn't fit well.